src/main/resources/db/migration/R__grant_to_cloudsqliamuser.sql

```diff
@@ -1,6 +1,4 @@
 REVOKE ALL ON ALL TABLES IN SCHEMA public FROM cloudsqliamuser;
 GRANT SELECT ON ALL TABLES IN SCHEMA public TO cloudsqliamuser;
 GRANT SELECT ON ALL TABLES IN SCHEMA public TO "isyfo-analyse";
-GRANT SELECT, INSERT, UPDATE, DELETE ON MOTEDELTAKER_BEHANDLER_VARSEL_SVAR TO cloudsqliamuser;
-GRANT SELECT ON MOTEDELTAKER_BEHANDLER_VARSEL_SVAR TO "isyfo-analyse";
```

Comment:
Hvordan fungerer det å endre en migration-fil i ettertid?

Reply:
Siden dette er repeatable så kan det endres. Flyway lagrer en sjekksum og kjører det på nytt hver gang det blir endret (det kjører til slutt etter at de vanlige scriptene er kjørt - så dette egner seg godt for å tilpasse tilgangskontrollen på tabellene).

Reply:
Okei, så bra 👍